Foundation/Data.swift

```diff
@@ -790,7 +790,7 @@ public final class _DataStorage {
     }
 }
 
-internal class _NSSwiftData : NSData {
+public class _NSSwiftData : NSData {
```

Comment:
this is quite unfortunate since Data.swift is shared tween swift-corelibs-foundation and the overlay